deploy/examples/dashboards/DashboardFromURL.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata: 
+  name: dashboard-from-url
+  labels: 
+    app: grafana
+spec: 
+  name: dashboard-from-url.json
+  url: https://gist.githubusercontent.com/jjaferson/e329d7cd6e806b52dbd98eaf67dca845/raw/3e8d6241439ea2b662e0fbf1d8e17acacb993647/grafana-dashboard-format.json
```

Comment:
I'd suggest adding a demo dashboard to the operator repo and referencing it from here

Reply:
I agree